astroid/brain/brain_namedtuple_enum.py

```diff
@@ -315,6 +315,7 @@ def infer_enum_class(node):
         if node.root().name == "enum":
             # Skip if the class is directly from enum module.
             break
+        dunder_members = {}
```

Comment:
I wonder if those modifications should not be the topic of another MR? 
Is there a corresponding issue that has been opened?
I can't see a strong relationship with other modifications of this MR.

Reply:
I answer to myself. It appears that PyCQA/pylint#4358 is the corresponding issue.

Reply:
@hippo91: sorry about the confusion, the `Enum.__members__` case was the original motivation for all of these changes (via https://github.com/PyCQA/astroid/pull/927#issuecomment-817138770 and above). I lost track of the issues that referenced this, I'll update the changelog and add a comment or so referencing them.

Reply:
@cdce8p: There were existing regression tests for `Enum.__members__` in pylint which passed due to `__members__` being uninferable (prior to #927 landing). I addressed one of them in this related PR: https://github.com/PyCQA/pylint/pull/4466. I can add additional regression tests that address the property/class-context case more directly

Reply:
> I can add additional regression tests that address the property/class-context case more directly

👍🏻 I would appreciate that

Reply:
Me too! :wink:

Reply:
@cdce8p, @hippo91: https://github.com/PyCQA/pylint/pull/4473